gravitee-apim-portal-webui-next/src/components/app-card/app-card.component.scss

```diff
@@ -19,9 +19,10 @@
   flex-flow: column;
 }
 
-.api-card {
+.app-card {
   display: flex;
   height: 100%;
+  min-height: 256px;
```

Comment:
This is ok, but it's best to avoid fixed heights in pixels for the CSS. If we ever want to create a mobile site (😱 ) or have a tablet-friendly site, it could make things difficult. If you'd like, we can 📞 to try to find a solution together. Otherwise, this can stay.

Reply:
Done